packages/expo-cli/src/commands/eject.ts

```diff
@@ -13,7 +13,7 @@ import { learnMore } from './utils/TerminalLink';
 
 async function userWantsToEjectWithoutUpgradingAsync() {
```

Comment:
I don't think I fully understand this feature. Many users eject because their project no longer runs in the client and they want to run it without needing to upgrade.

Reply:
yeah i think in those cases they can ignore it. we wouldn't force them to upgrade but rather strongly encourage them to do so